extensions/amp-story/1.0/amp-story.js

```diff
@@ -394,10 +395,6 @@ export class AmpStory extends AMP.BaseElement {
       'en-xa',
       enXaPseudoLocaleBundle
     );
-
-    registerServiceBuilder(this.win, 'localization', function () {
-      return localizationService;
-    });
```

Comment:
Any reason why we install it in core-services.js instead of here?

Reply:
Changed it so that it's installed in `localization.js` instead.